torch/quantization/quantize.py

```diff
@@ -23,8 +23,13 @@ def propagate_qconfig_helper(module, qconfig_dict, qconfig_parent=None, prefix='
     """
     if not hasattr(module, 'qconfig'):
         module.qconfig = None
-        if qconfig_dict and prefix in qconfig_dict:
-            module.qconfig = qconfig_dict[prefix]
+        if qconfig_dict:
+            if prefix in qconfig_dict:
+                module.qconfig = qconfig_dict[prefix]
+            elif type(module) in qconfig_dict:
+                module.qconfig = qconfig_dict[type(module)]
+            else:
```

Comment:
I think we don't need this, we can have `module.qconfig=qconfig_parent` in line 25 and remove all the else branches.

Reply:
Updated! Thanks @jerryzh168 !